controllers/example_controller/keyboard_controller.py

```diff
@@ -122,20 +134,20 @@ def print_sensors(robot: Robot) -> None:
             boost = True
 
         if key_ascii == key_forward:
-            left_power += 0.5
-            right_power += 0.5
+            left_power += 0.4
+            right_power += 0.4
 
         elif key_ascii == key_reverse:
-            left_power += -0.5
-            right_power += -0.5
+            left_power += -0.4
+            right_power += -0.4
 
         elif key_ascii == key_left:
-            left_power -= 0.25
-            right_power += 0.25
+            left_power -= 0.125
+            right_power += 0.125
 
         elif key_ascii == key_right:
-            left_power += 0.25
-            right_power -= 0.25
+            left_power += 0.125
+            right_power -= 0.125
```

Comment:
Please can we break this speed change out into a separate PR.

Reply:
removed these changes